frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/vm-settings-tab/workload-profile.tsx

```diff
@@ -110,6 +115,7 @@ export const WorkloadSelect: React.FC<WorkloadProps> = React.memo(
 type WorkloadProps = {
   iUserTemplate: any;
   commonTemplates: any;
+  additionalCommonTemplates: any;
```

Comment:
We should be using types we already have defined.
```suggestion
  additionalCommonTemplates: TemplateKind[];
```

Please update the template types above as well